packages/design-system/package.json

```diff
@@ -63,7 +63,7 @@
     "node-notifier": "^10.0.1",
     "sass": "^1.58.0",
     "sass-loader": "^13.2.0",
-    "storybook": "^7.0.0-beta.46",
+    "storybook": "^7.0.8",
     "storybook-addon-designs": "^6.3.1",
     "storybook-addon-themes": "^6.1.0",
```

Comment:
Does this addon work for you? Changing themes doesn't do anything on my setup, and I'm wondering if we should remove this addon.

Reply:
I see the class name toggled on a div but I can't see any changes

Reply:
Ok, it works (except on the introduction page)